include/dlr_treelite.h

```diff
@@ -23,6 +23,11 @@ struct TreeliteInput {
   size_t num_row;
   size_t num_col;
   CSRBatchHandle handle;
+
+  ~TreeliteInput() {
+    if (handle) TreeliteDeleteSparseBatch(handle);
```

Comment:
Does handle need to be initialized to nullptr in constructor?

Reply:
fixed. Looks like in C++11 we can just say `CSRBatchHandle handle = nullptr;`